Reqnroll.VisualStudio/Discovery/DiscoveryInvoker.cs

```diff
@@ -31,13 +31,14 @@ public ProjectBindingRegistry InvokeDiscoveryWithTimer()
     {
         var stopwatch = new Stopwatch();
         stopwatch.Start();
+        var projectSettings = _projectScope.GetProjectSettings();
```

Comment:
Why the need to modify this, non-test function?

Reply:
Guess small mistake while debugging/getting the tests working. Reverted!